.editorconfig

```diff
@@ -9,8 +9,8 @@ root = true
 # Use tabs as indentation
 # Trim trailing whitespace
 [*]
-insert_final_newline = true
-indent_style = space
+insert_final_newline = false
+indent_style = tabs
```

Comment:
I know it's a bit of a heated debate in the programming world, but is there a reason to switch this to tabs instead of spaces? ^^'>

Reply:
> I know it's a bit of a heated debate in the programming world, but is there a reason to switch this to tabs instead of spaces? ^^'>

Aren't tabs the standard? And yes, managing whitespace using tabs is easier

Reply:
https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#layout-conventions
> Use the default Code Editor settings (smart indenting, four-character indents, **tabs saved as spaces**). For more information, see [Options, Text Editor, C#, Formatting](https://learn.microsoft.com/en-us/visualstudio/ide/reference/options-text-editor-csharp-formatting).

Although, I'm not sure myself. I personally use spaces because it's easier to indent comments after variable/constant declarations, like so:
```cs
const int SomeConstant = 3;        // A constant value of "3".
const int SomeOtherConstant = 5;   // A constant value of "5".
const int NinePlusTen = 21;        // The result of the arithmetic operation "9 + 10".
const int AValueMaybe3 = 6;        // A value that's possibly 3, with no guarantee.
```
The IDE can help with automatically indenting this when using tabs. However, tabs are also good, so I'm gonna dismiss this comment.

Reply:
I talked with a few other coredevs and we would prefer if we stay with spaces.